RecoEgamma/PhotonIdentification/plugins/PhotonIDValueMapProducer.cc

```diff
@@ -358,7 +367,7 @@ void PhotonIDValueMapProducer::fillDescriptions(edm::ConfigurationDescriptions&
 // Charged isolation with respect to the worst vertex. See more
 // comments above at the function declaration.
 float PhotonIDValueMapProducer::computeWorstPFChargedIsolation(const reco::Photon& photon,
-                                                               const edm::View<reco::Candidate>& pfCands,
+                                                               const std::vector<edm::Ptr<reco::Candidate>> pfCands,
```

Comment:
```suggestion
                                                               const std::vector<edm::Ptr<reco::Candidate>>& pfCands,
```
to avoid a vector copy.